packages/wallet-nextjs/pages/_app.tsx

```diff
@@ -35,7 +36,8 @@ function MyApp({ Component, pageProps }: AppProps) {
       new NightlyWalletAdapter(),
       new BitkeepWalletAdapter(),
       new TokenPocketWalletAdapter(),
-      new BloctoWalletAdapter({ network:WalletAdapterNetwork.Testnet })
+      new BloctoWalletAdapter({ network:WalletAdapterNetwork.Testnet }),
```

Comment:
I think it is better to align with other wallets and left out the default network by setting it to undefined

Reply:
This is from BloctoWalletAdapter. We do not change this. This is from the most recent main branch commit. 